### DISCUSSION TITLE
[求助]邦布丢失了

### DISCUSSION THREAD

**Discussion by Qijoten** [share121/inter-knot#620] [其他]
本人于8月10日上午十点于六分街丢失一只邦布，这是邦布照片，求助各位绳匠
![Screenshot_20240810_125032_tv danmaku bilibilihd_edit_295195549884057](https://github.com/user-attachments/assets/9fccfb0f-40b5-4bb8-a940-8ee40a175484)

**Comment by FrostPhoenix79** [comment]
嘶，等等，我好像看到过这只邦布……不过是在录像店附近，一群邦布凑在一起好像在商量什么事情。楼主你再等等，说不定它就自己回来了。
